packages/transaction-controller/src/utils.ts

```diff
@@ -105,6 +105,16 @@ export function validateTxParams(txParams: TransactionParams) {
       );
     }
   }
+
+  if (
```

Comment:
The new function or logic might benefit from a comment clarifying that we don't actually use or persist this value but this matches extension validation, and we don't want to stop validating and give callers a false sense of security providing random values.